db/migrate/20200312110645_create_monthly_sp_auth_counts.rb

```diff
@@ -0,0 +1,15 @@
+class CreateMonthlySpAuthCounts < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+
+  def change
+    create_table :monthly_sp_auth_counts do |t|
+      t.string :issuer, null: false
+      t.integer :ial, null: false, limit: 1
+      t.string :year_month, null: false
+      t.integer :user_id, null: false
+      t.integer :auth_count, default: 1, null: false
+    end
+    add_index :monthly_sp_auth_counts, %i[issuer ial year_month user_id], algorithm: :concurrently,
```

Comment:
Do we need to build the index concurrently index for a new table? I think we could drop that and allow this migration to run in a transaction.